### ISSUE TITLE
ctx fields like ctx.status should be input.ctx.status and not input.status

### ISSUE THREAD

**Issue by wplu** [infobloxopen/seal#81]
see comment in:
https://github.com/infobloxopen/seal/pull/80

**Comment by wplu** [comment]
@rchowinfoblox said:
```
Forgot to mention, input.ctx will be an array, so it should be input.ctx[_].status == "reserved", etc.
```

why is ctx an array?

**Comment by rchowinfoblox** [comment]
@daniel-garcia wanted to support object hierarchies, so wanted ctx to support array of multiple objects.

**Comment by wplu** [comment]
we need to raise a different issue for multiple hierarchy and discuss how types is defined and structure of ctx

**Comment by abynenkov-ib** [comment]
Note: seems fix is the same as for #83  issue, and it's waiting for review in #102
